packages/manager/modules/vps/src/dashboard/vps-dashboard.constants.js

```diff
@@ -12,10 +12,37 @@ export const NEW_RANGE_VERSION = '2019v1';
 export const VPS_2014_AUTO_MIGRATION_DATE = '18/07/2020';
 export const VPS_2014_EXPIRY_DATE = '27/07/2020';
 
+export const GUIDES_RTM = {
+  CZ: 'https://docs.ovh.com/cz/cs/',
+  DE: 'https://docs.ovh.com/de/',
+  ES: 'https://docs.ovh.com/es/',
+  FI: 'https://docs.ovh.com/fi/',
```

Comment:
Still needed ?

Reply:
yes, it is a placeholder but i replace it for the right link when i got it.

Reply:
What I meant is do we still need to add urls for CZ, FI and LT, especially we removed FI from language menu earlier this year

Reply:
hum, i take reference in other part of code because i don't know each possibility. i remove it